### ISSUE TITLE
Plugin execution error based on dadaset.data start with the value "null"

### ISSUE THREAD

**Issue by whiteua** [Makanz/chartjs-plugin-trendline#23]
Hi, @Makanz!
When the dataset start with the value "null", the execution of the script is interrupted by error:
**Uncaught TypeError: Cannot read property 'x' of null**
chartjs-plugin-trendline.js: 52
Dataset example:
datasets: [{ 
     data: [null,1,1,2,3,5],
}]
chartjs-plugin-trendline v0.1.3
Chart.js v2.9.3

**Comment by whiteua** [comment]
I think the error is here:
```
var xy = false;
if ( dataset.data && typeof dataset.data[0] === 'object') xy = true;

```
Because:
`(typeof null === 'object') = true
`

**Comment by Makanz** [comment]
I think this problem is solved in the latest version v0.2.1
